definitions/json/AuthorisationCaseField.json

```diff
@@ -1,4 +1,34 @@
 [
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "scannedDocuments",	
+    "UserRole": "caseworker-ET-pcqextractor",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "ClaimantPcqId",	
+    "UserRole": "citizen",
+    "CRUD": "CRU"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "ClaimantPcqId",	
+    "UserRole": "caseworker-employment-etjudge",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "ClaimantPcqId",	
+    "UserRole": "caseworker-employment",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "pcqId",	
```

Comment:
"CaseFieldID": "pcqId",	 caseFieldId is referencing pcqId but it is not referencing in caseField.json. Is this a mistake.